kotlin/android/app/src/main/AndroidManifest.xml

```diff
@@ -51,7 +57,7 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
 
-                <data android:scheme="https" android:host="${hostName}" />
+                <data android:scheme="https" android:host="${hostName}" android:path="/handle_client_auth_callback" />
```

Comment:
```suggestion
                <data android:scheme="https" android:host="${hostName}" android:path="/handle_client_auth_callback" />
```

Without the `path` specifier here, the AppLink would intercept the initial `sign_in` path, preventing the user from actually signing in.